### ISSUE TITLE
no "unaligned.apk" file after successful (?) build

### ISSUE THREAD

**Issue by 255kb** [meteor/meteor#5274]
Hi, 

I updated my app to Meteor 1.2 and everything went fine. I also updated all cordova native plugins used by the app and switched to crosswalk.
I finally installed the android sdk through android studio. 
No other change to my dev environnement (Ubuntu linux 15.04 x64)
Running `meteor run android-device` works very well and I can successfully test the application. 

However, running `meteor build --directory --server=https://...` seems to run fine (no error message) but the file `unaligned.apk` is not present in the 'android' folder of the build directory.  Same with the `--debug` flag. 

Does anyone have the same problem?
Thanks

**Comment by martijnwalraven** [comment]
With Crosswalk added, there will actually be multiple apks for the different architectures. You can find them under `android/project/build/outputs/apk`. I'll make a note to document this somewhere.

**Comment by 255kb** [comment]
Thanks for the answer!
did not know about this, I thought it was some intermediate builds. 
But which one are we supposed to upload to the store ? All of them and the store take care to target the right platform?

**Comment by MoeBadr** [comment]
@255kb also this guide https://software.intel.com/sites/default/files/managed/9a/72/Intel_Walkthrough_GP-Multiple_APK_Submissions.pdf

**Comment by martijnwalraven** [comment]
@MoeBadr That's a great reference, thanks for posting it. I'll see if I can adapt this to the Meteor integration and add it to the Wiki.

**Comment by adamgins** [comment]
Hi folks, wondering if there's a new guide for this pls?

I was not sure if I should be doing something like:

```
 $ANDROID_HOME/build-tools/23.0.1/zipalign 4 android-armv7-release-unsigned.apk arm-production.apk

 $ANDROID_HOME/build-tools/23.0.1/zipalign 4 android-x86-release-unsigned.apk x86-production.apk

```

**Comment by 255kb** [comment]
yes, you have to first sign both files with `jarsigner` then align them with `zipalign` like you did before with `unalign.apk`. Just replace the files names.

**Comment by adamgins** [comment]
thanks

**Comment by VioletaCalvo** [comment]
Hi,

Thanks a lot for posting this!
I think this is a very important thing to appear in the documentation. I couldn't understand why there wasn't an .apk without build errors.

**Comment by kartikTT** [comment]
I am attempting to sign both apk files in the output directory, but I keep getting: error: java.lang.RuntimeException: keystore load: Invalid keystore format

I have confirmed that the password is correct and that the keystore exists with keytool -list -v. I am manually specifying the location of the keystore to the root directory of my app. 

Anyone successfully sign these apks?

**Comment by VioletaCalvo** [comment]
@kartikTT I signed these apks successfully, without errors, but I don't know how to help you, sorry.

**Comment by nombx** [comment]
@martijnwalraven  Please view this post https://github.com/meteor/meteor/issues/5833
